npm/create-cypress-tests/src/index.ts

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env node
+import { program } from 'commander'
+import { main } from './main'
+import { version } from '../package.json'
+
+program
+.option('--ignore-examples', 'Ignore generating example tests and fixtures by creating one ready-to-fill spec file')
```

Comment:
It is not lcear what this option do. I think we could find a better name for it 
Does it create an example?
Does it make an empty spec instead?